src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -906,6 +911,608 @@ public void SetAndInitAccessor()
             }
         }
 
+        [Theory, CompilerTrait(CompilerFeature.RequiredMembers)]
```

Comment:
Just curious, how do you make use of the trait attributes for test runs when developing with vscode?

Reply:
I don't. But I do make use of them if I happen to be on Windows.